components/minio.libsonnet

```diff
@@ -2,15 +2,15 @@
   local minio = self,
 
   config:: {
-    namespace: error 'must provide namespace',
+    s3namespace: error 'must provide namespace',
```

Comment:
this is a nested config already, there is no need for a separate name

Reply:
Done